cron/hourly/git_maintenance_tasks_lock

```diff
@@ -4,4 +4,5 @@ set -e
 # Run git_maintenance_tasks only if not running already
 DIR="$(readlink -e "$(dirname $0)/..")"
 mkdir -p ~/lmms-repo
-flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock "$DIR/git_maintenance_tasks"
+flock -w 0 ~/lmms-repo/git_maintenance_tasks.lock \
+	"$DIR/git_maintenance_tasks_retry"
```

Comment:
>If we abolish flock, then we don't need to turn trivial rube-goldberg solutions into 3, 4 or 5 script solutions.

@tresf, the use of `flock` has nothing to do with having 3, 4 or 5 scripts.

>That is matter of opinion.

Try to do it if you do not believe me.

>This is nice when we're looking for singleton.

No, it is not, because you cannot have another instance with the same process name.

>watching a script call flock, which calls a wrapper around a script which calls a script is also complex.

Using separate files is a matter of organization. Of course, you can have the whole procedure in a single file, but this is not maintainable. Do you want to merge the scripts?